definitions/variable/macro-economy/trade.yaml

```
@@ -8,3 +8,52 @@
     unit: billion USD_2010/yr
     tier: 1
     min: 0
+
+# emissions allowances
+- Trade|Emissions Allowances [Volume]:
+    description: Net trade in GHG emission allowances measured in physical quantities
```

While I would prefer "Net Exports" instead of "Trade" also here, I think at least the description should indicate this. Same below
```suggestion
    description: Net exports in GHG emission allowances measured in physical quantities
```

---

I don't think exports of allowances makes sense so I instead added a slightly longer description to add clarity.